bricks/template/__brick__/{{project_name.snakeCase()}}/lib/main.dart

```diff
@@ -4,6 +4,7 @@ import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:{{project_name.snakeCase()}}/gen/assets.gen.dart';
 import 'package:go_router/go_router.dart';
 import 'package:package_info_plus/package_info_plus.dart';
+import 'package:{{project_name.snakeCase()}}/gen/colors.gen.dart';
```

Comment:
could you please reorder the import to below of the `import 'package:{{project_name.snakeCase()}}/gen/assets.gen.dart';`?

Reply:
Fixed in https://github.com/nimblehq/flutter-templates/pull/231/commits/be37d7314d53fd11aadbe897625eadcc86c9e40e